blog/2021-05-16-terasaturday-14-spring-cleanup/index.md

```diff
@@ -29,7 +29,7 @@ While we keep tracking technical discussion on Github, the org meeting allows fo
 
 <div align="center">
     <a href="https://trello.com/b/4cw7Q1BJ/org-meeting-board" target="_blank" alt="Trello Org Meeting Board">
-    <img src="./trello_done_cards.png" alt="Weekly list of completed Trello cards." width="100%"/>
```

Comment:
lets keep this as PNG to preserve transparency (they are roughly the same size or even smaller anyways).